Packs/PassiveTotal/Integrations/PassiveTotal_v2/PassiveTotal_v2.yml

```diff
@@ -934,7 +1114,331 @@ script:
     - contextPath: PassiveTotal.Domain.techTelephone
       description: The telephone number of the domain tech.
       type: String
-  dockerimage: demisto/python3:3.8.6.12176
+  - arguments:
+    - default: false
+      description: 'Indicator value to search for in articles. For example: riskiq.com,
+        1.1.1.1'
+      isArray: false
+      name: query
+      required: true
+      secret: false
+    - default: false
+      description: 'Type of the indicator. For example: domain, ip, url'
+      isArray: false
+      name: type
```

Comment:
can we add predefined values here?

Reply:
@Itay4 we had no fixed set of indicator types in PassiveTotal API Doc so we decided to keep it is as it is after confirming it with the customer.